lib/properties.dart

```diff
@@ -20,13 +20,16 @@ class Properties {
   static final int EQUAL = '='.codeUnits[0];
 
   /// The content of the properties file in terms of key - value couples
-  Map<String, String> _content;
+  late Map<String, String?> _content;
 
   /// Layout manager
-  PropertiesLayout _layout;
+  PropertiesLayout? _layout;
 
   /// An internal reference to the source file.
-  String _sourceFile;
+  String? _sourceFile;
+
+  /// An internal reference to the String value.
+  String? _stringValue;
```

Comment:
we don't need this, actually - what about removing and inlining `_initFromString()`?
```suggestion
```

Reply:
I thought later it can be used for `toFile()`. I didn't understand `toFile()` method. This method can only be used if properties are loaded from file otherwise this method will throw exception.